### PR TITLE
Update slot definition and remove DuckDB driver cpuThreadRatio

### PR DESCRIPTION
**Pull request by kaspersjo**

### CODE REVIEW

File: runtime/drivers/duckdb/config.go [hunk 1]
```diff
@@ -104,7 +103,7 @@ func newConfig(cfgMap map[string]any) (*config, error) {
 	if cfg.ThreadsOverride != 0 {
 		threads = cfg.ThreadsOverride
 	} else if cfg.CPU > 0 {
-		threads = int(cpuThreadRatio * float64(cfg.CPU))
+		threads = cfg.CPU
 		if threads <= 0 {
```
Comment (k-anshul) [inline runtime/drivers/duckdb/config.go:107]: Can also remove this check fractional CPU is no longer possible now which would get translated to 0 previously ?

Comment (kaspersjo) [reply runtime/drivers/duckdb/config.go]: True - removed

**Comment by himadrisingh** [comment]
Lets keep it 1:4 for standard nodes. Highmem is costlier and will require more efforts in order to change the slots of all the current projects.

File: admin/provisioner/kubernetes.go [hunk 2]
```diff
@@ -123,7 +123,7 @@ func (p *KubernetesProvisioner) Provision(ctx context.Context, opts *ProvisionOp
 		Names:        names,
 		Host:         strings.Split(host, "//")[1], // Remove protocol
 		CPU:          1 * opts.Slots,
-		MemoryGB:     2 * opts.Slots,
+		MemoryGB:     8 * opts.Slots,
```
Comment (himadrisingh) [inline admin/provisioner/kubernetes.go:126]:
```suggestion
		MemoryGB:     4 * opts.Slots,
```

File: admin/provisioner/static.go [hunk 3]
```diff
@@ -72,7 +72,7 @@ func (p *StaticProvisioner) Provision(ctx context.Context, opts *ProvisionOption
 		Host:         target.Host,
 		Audience:     target.Audience,
 		CPU:          1 * opts.Slots,
-		MemoryGB:     2 * opts.Slots,
+		MemoryGB:     8 * opts.Slots,
```
Comment (himadrisingh) [inline admin/provisioner/static.go:75]:
```suggestion
		MemoryGB:     4 * opts.Slots,
```

**Review by himadrisingh — APPROVED** [review]

**Pull request merged**